src/components/Courses/components/CourseCard/coursecard.scss

```
@@ -0,0 +1,90 @@
+.card {
+    width: 80%;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    align-content: center;
+
+    background: #FFFFFF;
+    
+    border: 1px solid rgba(0, 0, 0, 0.12);
+    box-shadow: 10px 0px 0px -5px rgba(0, 0, 0, 0.8) inset, 4px 4px 3px rgba(0, 0, 0, 0.2), 0px -8px 5px -5px rgba(0, 0, 0, 0.2);
```

Unit of measure 'px' is redundant
```suggestion
    box-shadow: 10px 0 0 -5px rgba(0, 0, 0, 0.8) inset, 4px 4px 3px rgba(0, 0, 0, 0.2), 0 -8px 5px -5px rgba(0, 0, 0, 0.2);
```

---

thanks, refactored